pkg/controllers/gateway/gateway_controller.go

```diff
@@ -149,7 +172,7 @@ func (r *GatewayReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 
 // Configures Gateway tls & dns for each cluster it targets.
 // Returns the programmed status, a list of clusters that were programmed, if the gateway should be requeued, and any error
-func (r *GatewayReconciler) reconcileGateway(ctx context.Context, previous gatewayv1beta1.Gateway, gateway *gatewayv1beta1.Gateway) (metav1.ConditionStatus, []string, bool, error) {
+func (r *GatewayReconciler) reconcileGateway(ctx context.Context, previous gatewayv1beta1.Gateway, gateway *gatewayv1beta1.Gateway, params *Params) (metav1.ConditionStatus, []string, bool, error) {
```

Comment:
_ *Params?

Reply:
I think that given that we intend to use that parameter in the future it makes sense to leave it named. I'd use the `_` when it's a parameter that we don't care about (we don't yet, but we will)